src/os/bluestore/BitAllocator.cc

```diff
@@ -707,7 +707,7 @@ void BitMapAreaIN::init(int64_t total_blocks, int64_t area_idx, bool def)
   BitMapArea **children = new BitMapArea*[num_child];
   int i = 0;
   for (i = 0; i < num_child - 1; i++) {
-    if (m_level <= 2) {
+    if (m_level < 2) {
```

Comment:
Level 0 is zone, level 1 is leaf. So when this node is >= 2 ( non leaf and not zone) we create child as leaf. Looks correct to me